eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -111,7 +112,7 @@ trait Eclair {
 
 class EclairImpl(appKit: Kit) extends Eclair {
 
-  implicit val ec = appKit.system.dispatcher
+  implicit val ec: ExecutionContextExecutor = appKit.system.dispatcher
```

Comment:
nit: we really want that to be an `ExecutorContext`

Reply:
Interesting, I let IntelliJ suggest it, I'll update that.